pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[build-system]
+requires = [
+    "setuptools >= 65",
+    "setuptools_scm[toml] >= 6.2",
+    "setuptools_scm_git_archive",
+    "wheel >= 0.29.0",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "FusionDLS"
+description = "1D model for evaluating heat flux in tokamak SOL"
+readme = "README.md"
+authors = [
+    {name = "Cyd Cowley"},
+    {name = "Mike Kryjak"},
+]
+license = {file = "LICENSE"}
+dynamic = ["version"]
+keywords = ["analysis", "plasma", "research"]
+classifiers = [
+    "Programming Language :: Python",
+    "Development Status :: 3 - Alpha",
+    "Natural Language :: English",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+
+requires-python = ">=3.10"
+dependencies = [
+    "numpy >= 1.23",
+    "matplotlib >= 3.3.4",
+    "scipy >= 1.10.1",
+    "netCDF4 >= 1.5.6",
+    "freeqdsk >= 0.3",
+    "pandas"
+]
+
+[project.optional-dependencies]
+docs = [
+    "sphinx >= 5.3",
+    "sphinx_autodoc_typehints >= 1.19",
+    "sphinx-book-theme >= 0.4.0rc1",
+    "myst_parser >= 0.14.0",
+]
+tests = [
+    "pytest >= 3.3.0",
+    "pytest-cov",
+]
+linting = [
+  "black",
+  "isort",
+  "flake8",
```

Comment:
Just a minor thing I noticed: a list of Ruff rule sets are provided below, but the standard linting tool here is `flake8`. Should this be `ruff`?